vnext/Chakra/ChakraExecutor.h

```diff
@@ -158,6 +158,8 @@ class ChakraExecutor : public JSExecutor
     std::unique_ptr<DebugProtocolHandler>& debugProtocolHandler, std::unique_ptr<DebugService>& debugService);
   static void CHAKRA_CALLBACK ProcessDebuggerCommandQueueCallback(void* callbackState);
   void ProcessDebuggerCommandQueue();
+  bool isPatchingRequired = true;
```

Comment:
This should default to false. I think this should be initialized in the cpp file.